packages/land/contracts/Land.sol

```diff
@@ -19,11 +19,13 @@ contract Land is LandBaseToken, OperatorFiltererUpgradeable {
     uint16 internal constant TOTAL_BASIS_POINTS = 10000;
 
     IRoyaltyManager private royaltyManager;
+    address private _owner;
     uint8 private initialized;
```

Comment:
Maybe not in this PR but can we use the same naming convention ?

```
uint8 private _initialized;
bool private _initializing;
```

Reply:
Identifier already declared in ERC721BaseToken contract
`bool internal _initialized;`